src/renderer/components/ft-settings-menu/ft-settings-menu.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <menu
+    class="settingsMenu"
+  >
+    <h2 class="header">
+      {{ $t('Settings.Settings') }}
+    </h2>
+    <a
+      v-for="(settingsSection) in settingsSections"
+      :id="settingsSection.type + '-link'"
+      :key="settingsSection.type + '-link'"
+      :href="'#' + settingsSection.type"
```

Comment:
As the in-app navigation uses the URL hash too, it might be better if we find a different way to scroll to the sections to avoid potential navigation issues.

In release builds: `app://bundle/index.html#/subscriptions`
In dev mode: `http://localhost:9080/index.html#/subscriptions`